hal/targets/cmsis/TARGET_Freescale/TARGET_MCU_K64F/TOOLCHAIN_GCC_ARM/MK64FN1M0xxx12.ld

```diff
@@ -352,6 +347,7 @@ SECTIONS
   }
 
   /* Initializes stack on the end of block */
+  __StackTop = ORIGIN(m_data_2) + LENGTH(m_data_2);
```

Comment:
Looks like the stack was moved to the end of RAM. This should increase the heap size so +1.

Also, the GCC specific tweaks can now be removed in RTX_CM_Lib.h:
```
#if defined(__GNUC__) && !defined(__CC_ARM)     /* GCC */
extern uint32_t __StackTop[];
#define INITIAL_SP            (__StackTop)
#else
#define INITIAL_SP            (0x20030000UL)
#endif
```